core/state/intra_block_state.go

```diff
@@ -774,10 +776,7 @@ func (sdb *IntraBlockState) CreateAccount(addr common.Address, contractCreation
 		previous = sdb.getStateObject(addr)
 	}
 
-	newObj, prev := sdb.createObject(addr, previous)
-	if prev != nil {
-		newObj.setBalance(&prev.data.Balance)
```

Comment:
Is the balance not transferred to the new object now?

Reply:
It's already transferred in createObject, so I just removed duplicated logic.

Reply:
Ok